_activities/visualization.md

```diff
@@ -2,10 +2,11 @@
 title: "Visualization"
 author: "Riccardo Maria Bianchi"
 layout: plain
-redirect_from: /activities/visualization.html
+redirect_from: 
+  - /workinggroups/visualization.html
 ---
 
-The Visualization Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. 
+The Visualization Activity Area and Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. 
```

Comment:
```suggestion
The Visualisation Activity Area and Working Group (WG) gathers experts, developers, users and all people from the Particle Physics community interested in interactive data visualization and event displays. 
```

Reply:
Thanks - it wasn't my mistake, but good to correct!